tests/model_monitoring/test_monitoring_api.py

```diff
@@ -65,3 +70,34 @@ def test_record_result_updates_last_request() -> None:
         db_mock.patch_model_endpoint.call_args.kwargs["attributes"]["last_request"]
         == datetime_mock.isoformat()
     ), "last_request attribute of the model endpoint was not updated as expected"
+
+
+@pytest.mark.parametrize(
+    "function",
+    [
+        {
+            "func": str(pathlib.Path(__file__).parent / "assets" / "application.py"),
+            "application_class": DemoMonitoringAppV2(param_1=1, param_2=2),
+        },
+        {
+            "func": str(pathlib.Path(__file__).parent / "assets" / "application.py"),
+            "application_class": "DemoMonitoringAppV2",
+            "param_1": 1,
+            "param_2": 2,
+        },
+    ],
+)
+def test_create_model_monitoring_function(function) -> None:
+    app = mlrun.model_monitoring.api._create_model_monitoring_function_base(
```

Comment:
I would add a short description just to explain what are we going to test here (kwargs when app is a string and kwargs when app is an app class instance).